include/boost/numeric/ublas/tensor/static_strides.hpp

```diff
@@ -288,15 +291,35 @@ struct basic_static_strides<basic_static_extents<T,Extents...>, Layout>
 } // namespace boost::numeric::ublas
 
 namespace boost::numeric::ublas{
-    
-template <class L, class T, T... E>
-struct is_strides< basic_static_strides< basic_static_extents<T, E...>, L > > : std::true_type {};
-
-template <class T, T... E, class L>
-struct is_static< basic_static_strides< basic_static_extents<T, E...>, L > > : std::true_type {};
+      
+  template <class L, class T, T... E>
+  struct is_strides< basic_static_strides< basic_static_extents<T, E...>, L > > : std::true_type {};
+
+  template <class T, T... E, class L>
+  struct is_static< basic_static_strides< basic_static_extents<T, E...>, L > > : std::true_type {};
+
+  template <class T, T... E, class L>
+  struct is_static_rank< basic_static_strides< basic_static_extents<T, E...>, L > > : std::true_type {};
```

Comment:
I recommend to put the traits in a separate file.
This would simplify the specialization of those traits for different types.

Reply:
ok.